### DISCUSSION TITLE
Trying to receive orders data from different shops ( clinets ) from my commercial app ?

### DISCUSSION THREAD

**Discussion by abhilas94** [etsy/open-api#1149] [Q&A]
Trying to receive orders data from different shops ( clients ) for my commercial app ? I facing some problem. How the things are working. Can someone please guide me.

**Comment by Gareth064** [comment]
Hi.

Can you share what the problem is your facing.

Example code you are using.
What error message you are getting.
The curl command that generates the error etc.
